.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Based on the project requirement, the `test.md` should be included in this .gitignore file so it is ignored by git. But that was not done which is why I can see the test.md file in your GitHub Repo. To fix this, manually remove this `test.md ` file from your GitHub repository, then add it to the .gitignore file before you push it again to GitHub.

#### .gitignore file
```md
node_modules/
test.md
```